### DISCUSSION TITLE
Having an issue with installing backpack/devtools and any paid package

### DISCUSSION THREAD

**Discussion by ahmedshahd** [Laravel-Backpack/community-forum#942] [Q&A (Help)] [answered]
when i try to install a paid package i get this error 

<img width="881" alt="Screenshot 2024-04-16 at 10 46 20 AM" src="https://github.com/Laravel-Backpack/community-forum/assets/48631752/a47b0eea-fc52-4d45-a1d7-afe8578fe667">

i have auth.json file with this structure 
{
    "http-basic": {
        "backpackforlaravel.com": {
            "username": "username provided by pack back",
            "password": "password provided by pack back"
        }},
    "github-oauth": {
        "github.com": "my github personal token"
    }
}

and i added this to my composer.json
 "repositories": [
        {
            "type": "composer",
            "url": "https://repo.backpackforlaravel.com/"
        }
    ],

**Comment by karandatwani92** [comment, marked as answer]
Hey @ahmedshahd 

Can you visit https://backpackforlaravel.com/user/tokens and check the `Updates Until` column to confirm if you still have access to the latest version.
